### PR TITLE
Test model consistency

### PR DESCRIPTION
**Pull request by jimmyDunne**
Test that the API can read its own printed models

### CODE REVIEW

File: test_models.py [hunk 1]
```diff
@@ -23,13 +23,29 @@
...
     except Exception as e:
-        print("Oops, Model '%s' failed:\n%s" % (modelNames[i], e.message))
+        print("Oops, Model '%s' failed:\n%s" % (modelname, e.message))
+        sys.exit(1)
+
+    # Print the 4.0 version to file and then read back into memory
+    filename_new = filename.replace('.osim','_new.osim')
+    modelName_new = modelName.replace('.osim','_new.osim')
+    # Print the 4.0 model to file
+    model.print(filename_new)
```
Comment (chrisdembia) [inline test_models.py]:
```suggestion
    model.printToXML(filename_new)
```

Comment (aseth1) [reply test_models.py]: 👍

**Comment by jimmyDunne** [comment]
@chrisdembia, could you please test locally and make any final changes? I can't get my Python—opensim environment working to test.

File: test_models.py [hunk 2]
```diff
@@ -23,13 +23,33 @@
...
+        reloadedModel = opensim.Model(filename_new)
+        s2 = reloadedModel.initSystem()
+    except  Exception as e:
+        print("Oops, 4.0 written Model '%s' failed:\n%s" % (modelName_new, e.message))
+        sys.exit(1)
+
+    # Remove the printed file
+    os.remove(filename_new)
+
+    if ~reloadedModel.isEqualTo(model)
```
Comment (chrisdembia) [inline test_models.py]:
```suggestion
    if not reloadedModel.isEqualTo(model):
```

File: test_models.py [hunk 3]
```diff
@@ -23,13 +24,33 @@
...
     except Exception as e:
-        print("Oops, Model '%s' failed:\n%s" % (modelNames[i], e.message))
+        print("Oops, Model '%s' failed:\n%s" % (modelname, e.message))
         sys.exit(1)
-        
-print("All models loaded successfully.")
 
+    # Print the 4.0 version to file and then read back into memory
+    filename_new = filename.replace('.osim','_new.osim')
+    modelName_new = modelName.replace('.osim','_new.osim')
```
Comment (chrisdembia) [inline test_models.py]: This might need to be `modelname` with a lower-case `n`.

Comment (chrisdembia) [reply test_models.py]: That's why the AppVeyor build failed.

**Comment by chrisdembia** [comment]
@aseth1's comment is addressed. Merging.

**Pull request merged**